src/components/SpreadTheWordButton/__snapshots__/index.test.js.snap

```diff
@@ -25,6 +25,7 @@ exports[`SpreadTheWordButton renders correctly 1`] = `
         >
           <svg
             aria-hidden="true"
+            className=""
```

Comment:
why are there all these changes to snapshots in this test that are unrelated?

Reply:
Same thing happens to me all the time. If you run all tests some unrelated snapshots change. This is started to happen since yesterday I believe.